builtin/providers/scaleway/resource_scaleway_server.go

```diff
@@ -70,6 +71,13 @@ func resourceScalewayServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"volumes": &schema.Schema{
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeInt,
+				},
+				Required: true,
```

Comment:
When you don't specify volumes you'll just get the default root volume after invoking `api.PostServer()`